practice/src/css/components/input.css

```diff
@@ -0,0 +1,15 @@
+.input-form {
+  margin-bottom: 10px;
+  border: 1px solid var(--tertiary-border-color);
+  border-radius: 8px;
+  padding: 14px;
+  background-color: var(--tertiary-background-color);
+  font-size: var(--font-size-xs);
+  font-weight: var(--font-weight-regular);
+  line-height: var(--line-height-lg);
+  color: var(--primary-text-color);
+}
+
+.input-quantity {
```

Comment:
Should create a common class here. Such as .input-form-lg, .input-form-sm. You can refer https://getbootstrap.com/docs/3.4/css/#forms-control-sizes